### PR TITLE
Make 'View All' search through all panes

### PR DESCRIPTION
**Pull request by acontreras89**
Addresses atom/bookmarks#35.

### CODE REVIEW

**Comment by 50Wliu** [comment]
Can you please add specs to this?

**Comment by acontreras89** [comment]
There you go, @50Wliu 
Let me know if you need anything else

File: spec/bookmarks-view-spec.coffee [hunk 1]
```diff
@@ -200,3 +200,22 @@ describe "Bookmarks package", ->
...
+        editor.setCursorBufferPosition([8])
+        atom.commands.dispatch editorElement, 'bookmarks:toggle-bookmark'
+        editor.setCursorBufferPosition([0])
+
+        pane1 = atom.workspace.getActivePane()
+        pane1.splitRight()
+        expect(atom.workspace.getActivePane()).not.toEqual pane1
+
+        atom.commands.dispatch workspaceElement, 'bookmarks:view-all'
+        $(workspaceElement).find('.bookmarks-view .bookmark').mousedown().mouseup()
```
Comment (benogle) [inline spec/bookmarks-view-spec.coffee]: Can you use `querySelector` rather than jquery?

**Comment by acontreras89** [comment]
Ready @benogle !

**Comment by benogle** [comment]
Thanks!

**Comment by acontreras89** [comment]
sweet! :dancer:

**Pull request merged**